### DISCUSSION TITLE
Critical RaspberryPi4+USB3+GPIO use detected

### DISCUSSION THREAD

**Discussion by teppic01** [OpenCCU/OpenCCU#1984] [🇩🇪 Fragen&Antworten] [answered]
Hallo Zusammen,
seit dem letzten Fix bekomme ich ständig diesen "Hinweis" angezeigt. Den Hintergrund habe ich mir inzwischen angelesen. Allerdings läuft bei mir HA auf einem PI400. Da ich eine SSD und einen Zigbee Stick angeschlossen habe, kann ich nicht auf USB2 umstecken, weil nur ein Anschluss vorhanden ist. Bei mir hängt das Funkmodul auch abgesetzt in einem Gehäuse mit einer separaten Antenne. Das Problem kann also nicht auftreten.
Daher meine Frage. Kann man den Hinweis irgendwie dauerhaft abstellen? 
Dazu konnte ich leider nichts finden.
Gruß
T.

**Comment by MichaelN0815** [comment]
Siehe erweiterte Einstellungen

**Reply by teppic01** [reply]
Super! :-)
Ich danke dir vielmals! 
Gruß
T.

**Comment by jens-maus** [comment, marked as answer]
Der vollständig halber hier noch der eindringliche Hinweis das das pure abschalten dieser kritischen Warnung nicht der 1. Weg bzw. empfehlenswert ist. Es gibt einen eindringlichen Grund wieso man diese Warnung ernst nehmen sollte. Der Pi4 ist und bleibt ein starker Funkstörer an dem man (wenn man ein Funkmodul lokal am Pi4 nutzt) keinerlei Dinge an den USB3 Bus (die blauen USB Buchsen) anschließen sollte weil es sonst zu massiven Funkstörungen kommt!

**Comment by Mab306** [comment]
Gibt es schon eine lösung für das oben genannte Problem? Ich nutze eine SSD an dem USB Port und ein HM-MOD-RPI-PCB leider bekomme ich auch diese Fehlermeldung! Bitte dringend um hilfe

**Reply by MichaelN0815** [reply]
Verrückte Idee : wie wäre es erstmal den kompletten Thread zu lesen?

**Reply by Mab306** [reply]
Und wie ? bin ziemlich neu ich verstehe noch nicht alles hier. Kannst mir auch die lösung senden ....

**Reply by MichaelN0815** [reply]
In den erweiterten Einstellungen (Systemsteuerung) die Warnung deaktivieren, wenn du dir über die Folgen klar bist. USB 3 kann zu Störungen des homematic Funk führen. Unbedingt CarrierSense auf der Startseite prüfen.

Um es noch mal ganz klar zu sagen : Die Meldung ist nicht das Problem, sondern die Nutzung der USB 3 Schnittstelle.

**Comment by fcmircea** [comment]
Hello, I have the same error and I would like it disabled / ignored.

- I have an antenna extension which places it well far from the pi4 (3m at least)
- I do have a SSD connected via USB3, but there are no connection / speed issues with it in dmesg.
- Carrier sense is at most 3%
- I have 2 devices conencted and I can operate them without issue (smart thermostat valve and smart socket).

Where in advanced settings can I disable this error?

Thanks!

**Comment by teppic01** [comment]
Hi,
easy to find i think... 
Advanced Settings / Watch Dog Options / USB3 Check
But this only disables the Error Message... use with care!

**Comment by fcmircea** [comment]
I am sorry, I must be missing something.

I'm running raspberrymatic as a home assistant add-on.

I don't seem to find the watchdogs options anywhere in the raspberrymatic
interface.

Should I be looking elsewhere?

Thanks


[image: image.png]


În dum., 18 feb. 2024 la 20:39, teppic01 ***@***.***> a scris:

**Comment by fcmircea** [comment]
Found it:

Settings -> Control Panel -> Advanced Settings -> Watchdog Options.

[image: image.png]


În dum., 18 feb. 2024 la 21:12, Florin-Constantin Mircea <
***@***.***> a scris:
